cli/azd/internal/cmd/add/add.go

```diff
@@ -300,21 +367,23 @@ func (a *AddAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			return nil, err
 		}
 
-		return &actions.ActionResult{
-			Message: &actions.ResultMessage{
-				FollowUp: "Run '" +
-					color.BlueString(fmt.Sprintf("azd show %s", resourceToAdd.Name)) +
-					"' to show details about the newly provisioned resource.",
-			},
-		}, nil
+		followUpMessage = "Run '" +
+			color.BlueString(fmt.Sprintf("azd show %s", resourceToAdd.Name)) +
```

Comment:
Nit: We have predefined format for common colors used in the CLI.  This should likely use `output.WithHighlightFormat(...)`

https://github.com/Azure/azure-dev/blob/main/cli/azd/pkg/output/colors.go#L18

Reply:
Thanks, I'll address this in https://github.com/Azure/azure-dev/pull/4847